be/src/exec/pipeline/exchange/sink_buffer.h

```diff
@@ -31,6 +34,7 @@ class SinkBuffer {
 public:
     SinkBuffer(MemTracker* mem_tracker, const std::vector<TPlanFragmentDestination>& destinations, size_t num_sinkers)
             : _mem_tracker(mem_tracker) {
+        _io_threads = ExecEnv::GetInstance()->pipeline_sink_io_thread_pool();
```

Comment:
pipeline_sink_io_thread_pool is confusing, suggest to use pipeline_exchange_sink_thread. io here is ambiguous with disk io.

Reply:
ok